hazelcast/src/main/java/com/hazelcast/config/IndexConfig.java

```diff
@@ -90,6 +92,7 @@ public IndexConfig(IndexConfig other) {
         this.name = other.name;
         this.type = other.type;
         this.bitmapIndexOptions = other.bitmapIndexOptions == null ? null : new BitmapIndexOptions(other.bitmapIndexOptions);
```

Comment:
imho for consistency bitmapIndexOptions -> bitmapIndexConfig or breeIndexConfig -> btreeIndexOptions. I've chosen BTreeIndexConfig as it's most consistent with other configuration options, but I can change it at any time.

(maybe rename bitmapIndexOptions -> bitmapIndexConfig and make backward-compatibility changes?)

CC @petrpleshachkov @taburet @frant-hartm